.github/workflows/e2e.yml

```diff
@@ -91,9 +96,8 @@ jobs:
             - name: Install python dependencies
               if: steps.cache-virtualenv.outputs.cache-hit != 'true'
               run: |
-                  python -m pip install --upgrade pip
-                  python -m pip install $(grep -ivE "psycopg2" requirements.txt | cut -d'#' -f1) --no-cache-dir --compile
-                  python -m pip install psycopg2-binary --no-cache-dir --compile
+                  python -m pip install -r requirements-dev.txt
```

Comment:
Should we still have `--no-cache-dir` here?

Reply:
ah, possibly :)

this workflow can't install the infi.clickhouse_orm package but the others can... was just hitting it with a stick to see what happens

Reply:
the backend workflow does

```
            - name: Install python dependencies
              if: steps.cache-virtualenv.outputs.cache-hit != 'true'
              run: |
                  python -m pip install -r requirements-dev.txt
                  python -m pip install -r requirements.txt
```

which is working in that workflow 🤷